File: src/main/resources/js/app/AppService.js
'use strict'

angular.module('demo.services', []).factory('AppService',
		[ "$http", "CONSTANTS", function($http, CONSTANTS) {
			var service = {};
			service.getApplicantById = function(id) {
				var url = CONSTANTS.getApplicantById + id;
				return $http.get(url);
			}
			service.getAllUsers = function() {
				return $http.get(CONSTANTS.getAllUsers);
			}
			service.saveUser = function(userDto) {
				return $http.post(CONSTANTS.saveUser, applicantDto);
			}
			return service;
		} ]);

File: src/main/java/com/flow/enginehouse/service/AppService.java
package com.flow.enginehouse.service;

import java.util.List;
import java.util.Map;
import com.flow.enginehouse.dto.ApplicantDto;

public interface AppService {
    ApplicantDto getApplicantById(Long id);
    Map<String,Object> saveUser(ApplicantDto appDto);
    List<ApplicantDto> getAllUsers();
}
